DQM/BeamMonitor/plugins/AlcaBeamMonitor.cc

```diff
@@ -296,7 +296,13 @@ void AlcaBeamMonitor::analyze(const Event& iEvent, const EventSetup& iSetup) {
   //------ Primary Vertices
   Handle<VertexCollection> PVCollection;
   if (iEvent.getByToken(primaryVertexLabel_, PVCollection)) {
-    beamSpotInfo->vertices_.push_back(*PVCollection.product());
+    std::vector<alcabeammonitor::pvPosAndErr> vertices;
```

Comment:
how about `vertices.reserve(PVCollection->size())`

Reply:
thanks included in the last push.